test/jsonapi.js

```diff
@@ -14,6 +14,9 @@ import {
   IS_UPDATING
 } from '../src/jsonapi';
 
+const fetchMock = require('fetch-mock');
```

Comment:
addressed in most recent commit [be37aa3](https://github.com/dixieio/redux-json-api/pull/61/commits/be37aa3d5f004bfc73885551c914bd41b4605f21)